### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by silvestreh**
### Summary

> - [ ] Tell us about the problem your pull request is solving.

Just thought of upgrading the project's dependencies and make all tests remain ✅

> - [ ] Are there any open issues that are related to this?

#358 is at least one.

> - [ ] Is this PR dependent on PRs in other repos?

Nope

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -40,7 +40,8 @@ class Service extends AdapterService {
     const whitelist = Object.keys(operators).concat(options.whitelist || []);
     const { primaryKeyAttributes } = options.Model;
     const id = typeof primaryKeyAttributes === 'object' && primaryKeyAttributes[0] !== undefined
-      ? primaryKeyAttributes[0] : 'id';
+      ? primaryKeyAttributes[0]
+      : 'id';
```
Comment (silvestreh) [inline lib/index.js:44]: These changes were required by the linter.

File: .mocharc.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "timeout": 10000,
+  "recursive": "test/",
+  "exit": true
+}
```
Comment (silvestreh) [inline .mocharc.json:5]: When upgrading mocha I had to create the rc file because the good old `.opts` format is no longer supported

**Comment by joshuaja** [comment]
Has this been reviewed yet? It would be awesome to use a newer version of `sequelize`

**Comment by daffl** [comment]
Sequelize is only a devDependency for this adapter - since there are no breaking changes you just have to update the Sequelize dependency in your app.

**Comment by joshuaja** [comment]
> Sequelize is only a devDependency for this adapter - since there are no breaking changes you just have to update the Sequelize dependency in your app.

Awesome, thanks so much for clarifying!

**Pull request merged**